packages/bpk-component-modal/src/BpkModal.js

```diff
@@ -33,14 +33,20 @@ const BpkModal = (props) => {
     onClose,
     target,
     renderTarget,
+    hideBackgroundOnMobile,
```

Comment:
I think that this behaviour should be encapsulated in a `fullScreenOnMobile` prop.